docs/src/appInsights.ts

```diff
@@ -0,0 +1,22 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+// eslint-disable-next-line import/no-unresolved
+import siteConfig from "@generated/docusaurus.config";
+import { ReactPlugin } from "@microsoft/applicationinsights-react-js";
+import { ApplicationInsights } from "@microsoft/applicationinsights-web";
+const reactPlugin = new ReactPlugin();
+
+const appInsights = new ApplicationInsights({
+	config: {
```

Comment:
We should set up the website so that local development skips all of this entirely. Right now it looks like it might be sending telemetry with a bad app id / instrumentation key?